contrib/examples/actions/echochain.meta.yaml

```diff
@@ -10,4 +10,11 @@ runner_type: "action-chain"
 entry_point: "chains/echochain.yaml"
 
 enabled: true
-parameters: {}
+parameters:
+  skip_notify:
```

Comment:
Does it? I probably missed this. It used to be that CLI didn't parse a comma separated list as an array.